controllers/machine_controller_phases_test.go

```diff
@@ -206,18 +206,21 @@ var _ = Describe("Reconcile Machine Phases", func() {
 		}, "status", "addresses")
 		Expect(err).NotTo(HaveOccurred())
 
+		// Set NodeRef.
```

Comment:
I can't comment on the line in question, but the description for this `It` case needs to be updated since we're expecting `Running` now

Reply:
sorry missed that one, fixed now